torch/csrc/lazy/ts_backend/tensor_aten_ops.cpp

```diff
@@ -20,6 +19,7 @@
 #include <torch/csrc/lazy/core/view_ops/unsqueeze.h>
 #include <torch/csrc/lazy/core/view_ops/view.h>
 #include <torch/csrc/lazy/generated/LazyIr.h>
+#include <torch/csrc/lazy/ts_backend/ops/random_ops.h>
```

Comment:
@antoniojkim Why were some ops left behind in `ts_backend/ops`? Are these TS specific?

Reply:
Yes, they have custom lowering rules defined. However, after discussion with @wconstab I think what might end up happening is that we'll make all of those ops (including the ones remaining in `ts_backend` inherit from `Node` itself and then make TS specific versions of those ops that inherit from both their `Node` subclass counterparts and from `TsNode`.